test/spec/bidmanager_spec.js

```diff
@@ -465,7 +465,9 @@ describe('bidmanager.js', function () {
     before(() => {
       $$PREBID_GLOBAL$$.adUnits = fixtures.getAdUnits();
     });
-
+    after(() =>{
+      $$PREBID_GLOBAL$$.adUnits = [];
```

Comment:
not sure of the point of setting `adUnits` to empty array just before it will be set to `fixtures.getAdUnits();` above? Basically the `before` does the reset.